pkg/codegen/dotnet/gen_program.go

```diff
@@ -346,6 +346,47 @@ func (g *generator) makeResourceName(baseName, count string) string {
 	return fmt.Sprintf("$\"%s-{%s}\"", baseName, count)
 }
 
+func (g *generator) genResourceOptions(opts *hcl2.ResourceOptions) string {
+	if opts == nil {
+		return ""
+	}
+
+	var result bytes.Buffer
+	appendOption := func(name string, value model.Expression) {
+		if result.Len() == 0 {
+			_, err := fmt.Fprint(&result, ", new CustomResourceOptions {")
```

Comment:
We tend to put `{` on a new line in the generated code

Reply:
Good point; I'll fix this up.